File: GHL_Revamp_Dev/Calendars/get_calendar.js
const axios = require('axios');
const config = require("../environment/config");
const constant = require("../environment/constant");



const headers = {
    "Authorization": `Bearer ${config.Nestle_access_token}`,
    "Version": "2021-07-28",
    "Content-Type": "application/json",
    "Accept": "application/json"
};

axios.get(config.calendars_url+constant.calendar_id1,{ headers })
.then(response => {
    console.log(response.data)
})
.catch(error => {
    if (error.response) {
        console.error('Error Response:', {
            status: error.response.status,
            statusText: error.response.statusText,
            data: error.response.data,
        });
    } else if (error.request) {
        console.error('No Response Received:', error.request);
    } else {
        console.error('Request Error:', error.message);
    }
});

File: GHL_Revamp_Dev(Python)/Calendars/get_calendar.py
import requests
from environment import config, constant


headers = {
    "Authorization": f"Bearer {config.config.Nestle_access_token}",
    "Version": "2021-07-28",
    "Content-Type": "application/json",
    "Accept": "application/json"
}

response = requests.get(f"{config.config.calendars_url}{constant.constant.calendar_id1}", headers=headers)

if response.status_code == 200:
    print(response.json())
else:
    print(f"Error: {response.status_code}")
    print(response.json())